config.json

```diff
@@ -22,7 +22,7 @@
     "solution": ["Sources/%{pascal_slug}/%{pascal_slug}.swift"],
     "test": ["Tests/%{pascal_slug}Tests/%{pascal_slug}Tests.swift"],
     "example": ["Sources/%{pascal_slug}/%{pascal_slug}Example.swift"],
-    "exemplar": [".meta/Exemplar.swift"]
+    "exemplar": [".meta/ExemplarSources/%{pascal_slug}.swift"]
```

Comment:
Was there any particular reason not yo use `.meta/Sources` like the Practice Exercises?

Reply:
Not really, I mainly wanted to make it explicit that the folder contained an exemplar implementation, since the file name no longer contained the `Exemplar` term. 

I vaguely recall that I read in the documentation that a concept exercise could contain more than one example implementation, which would make the directory naming more important, but reading through it again I can’t make out where I got that idea.

I think using `.meta/Sources` like the practice exercises makes the most sense, so I’ll push an update to fix it.

Reply:
Great! There can only be one exemplar solution (it can consist of multiple files though).

Reply:
I just noticed that the practice exercises actually use `.meta/Sources/%{pascal_slug}/%{pascal_slug}Example.swift`, so I think the concept exercises might use that same pattern as well, so `.meta/Sources/%{pascal_slug}/%{pascal_slug}Exemplar.swift`.